### PR TITLE
Geomedian 4.0.0 - new geomedian version WIP

### PR DESCRIPTION
**Pull request by JM-GA**
Do not approve or merge this PR. Putting in PR to preview the visualisation of the changes only for testing purposes.

* [ ] I have spellchecked my content using Microsoft Word, Grammarly, or otherwise.
* [ ] If required, update the [DEA Tech Alerts and Changelog](TechAlertsChangelog) and the [DEA Sandbox Updates banner](SandboxUpdatesBanner).

[TechAlertsChangelog]: https://github.com/GeoscienceAustralia/dea-docs/blob/main/docs/tech-alerts-changelog/_tech_alerts_changelog.md
[SandboxUpdatesBanner]: https://bitbucket.org/geoscienceaustralia/datakube-apps/src/64c28bbf3d0e019d8940547a22f78b9bfd58d739/clusters/dea-sandbox/sandbox.yaml

### CODE REVIEW

**Comment by benji-glitsos-ga** [comment]
**Pages:**

* https://pr-149-preview.khpreview.dea.ga.gov.au/data/product/dea-geometric-median-and-median-absolute-deviation-landsat/?tab=overview
* https://pr-149-preview.khpreview.dea.ga.gov.au/data/old-version/dea-geometric-median-and-median-absolute-deviation-landsat-3.1.0/?tab=overview

**Comment by JM-GA** [comment]
@benji-glitsos-ga , @CEKrause  - I have put in PR for tentative approval - but we will need approval first to push data to prod, to then update the access links and then we can release this

**Comment by benji-glitsos-ga** [comment]
@JM-GA I've done a final check and I'm happy to go ahead with final approvals.

**Comment by benji-glitsos-ga** [comment]
@JM-GA It's now possible to add 'recommended citations' to the product page. Here's an example from [DEA Coastlines](https://knowledge.dea.ga.gov.au/data/product/dea-coastlines/).

ksnip_20240404-115043

You can provide one or both citations in plain text format in the YAML like this:

```yaml
citations:
  data_citation: Bishop-Taylor, R., Nanson, R., Sagar, S., Lymburner, L. (2021). Digital Earth Australia Coastlines. Geoscience Australia, Canberra. https://doi.org/10.26186/116268
  paper_citation: Bishop-Taylor, R., Nanson, R., Sagar, S., Lymburner, L. (2021). Mapping Australia's dynamic coastline at mean sea level using three decades of Landsat imagery. Remote Sensing of Environment, 267, 112734. https://doi.org/10.1016/j.rse.2021.112734
```

**Comment by JM-GA** [comment]
Thanks @benji-glitsos-ga - sounds good! I will add a citation for the data

**Comment by benji-glitsos-ga** [comment]
@JM-GA I changed  '20 tiles west and 17 tiles south' to '18 tiles west and 15 tiles south' in accordance with this thread raised by Emma: https://teams.microsoft.com/l/message/19:7ffb43c0324c42cb887ef978f88e34b2@thread.tacv2/1716275830628?tenantId=645b613a-a64c-4250-975f-5f69787d028d&groupId=80293ccc-1a98-4155-88e4-c7f6bb6c942b&parentMessageId=1716275830628&teamName=DEA&channelName=General&createdTime=1716275830628

See the change: https://github.com/GeoscienceAustralia/dea-knowledge-hub/pull/149/files#diff-3bc263b183b3702db46418db2b5427751a12f78935ed88a9e5d016812f94d473

**Comment by benji-glitsos-ga** [comment]
**Preview:**

* https://pr-149-preview.khpreview.dea.ga.gov.au/data/product/dea-geometric-median-and-median-absolute-deviation-landsat/?tab=overview
* https://pr-149-preview.khpreview.dea.ga.gov.au/guides/reference/collection_3_summary_grid/
* https://pr-149-preview.khpreview.dea.ga.gov.au/tech-alerts-changelog/#in-progress-shift-in-origin-point-of-dea-summary-product-grid

File: docs/data/product/dea-geometric-median-and-median-absolute-deviation-landsat/_details.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ $$
 \end{align*}
 $$
 
-In GeoMAD, the MADs are calculated from the same ten bands used in the geometric median, therefore $p=10$. The result of $\lVert \mathbf{x}^{(t)} - \mathbf{m} \rVert_{\mathbb{R}^p}$ is a positive scalar, so $\text{EMAD}_\text{GeoMAD}$ is a positive scalar number. As in the geometric median, $N$ is dependent on the number of satellite flyovers particular to that pixel.
+In GeoMAD, the MADs are calculated from the same six bands used in the geometric median, therefore $p=6$. The result of $\lVert \mathbf{x}^{(t)} - \mathbf{m} \rVert_{\mathbb{R}^p}$ is a positive scalar, so $\text{EMAD}_\text{GeoMAD}$ is a positive scalar number. As in the geometric median, $N$ is dependent on the number of satellite flyovers particular to that pixel.
```
Comment (benji-glitsos-ga) [inline docs/data/product/dea-geometric-median-and-median-absolute-deviation-landsat/_details.md:218]:
```suggestion
In GeoMAD, the MADs are calculated from the same 6 bands used in the geometric median, therefore $p=6$. The result of $\lVert \mathbf{x}^{(t)} - \mathbf{m} \rVert_{\mathbb{R}^p}$ is a positive scalar, so $\text{EMAD}_\text{GeoMAD}$ is a positive scalar number. As in the geometric median, $N$ is dependent on the number of satellite flyovers particular to that pixel.
```

In scientific writing, I now try to use numerals instead.

Comment (robbibt) [reply docs/data/product/dea-geometric-median-and-median-absolute-deviation-landsat/_details.md:218]: There's a few different recommended approaches here I think - I was always taught to spell out numbers less than 10, and use numerals for anything greater than that ([AP/APA style here](https://www.masterclass.com/articles/when-to-spell-out-numbers-explained)) - I've seen spelling them out below 100 recommended too.

Comment (benji-glitsos-ga) [reply docs/data/product/dea-geometric-median-and-median-absolute-deviation-landsat/_details.md:218]: I did some research and actually scientific writing uses one to nine in worded form, as you mentioned. I read something recently that gave me the wrong idea.

File: docs/data/old-version/dea-geometric-median-and-median-absolute-deviation-landsat-3.1.0/_data.yaml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    name: Annual Landsat 5 observations on NCI
+  - link: https://dapds00.nci.org.au/thredds/catalog/jw04/ga_ls7e_nbart_gm_cyear_3/catalog.html
+    name: Annual Landsat 7 observations on NCI
+  - link: https://dapds00.nci.org.au/thredds/catalog/jw04/ga_ls8c_nbart_gm_cyear_3/catalog.html
+    name: Annual Landsat 8 observations on NCI
+
+code_examples: null
+
+web_services:
+  - link: https://ows.dea.ga.gov.au/
```
Comment (benji-glitsos-ga) [inline docs/data/old-version/dea-geometric-median-and-median-absolute-deviation-landsat-3.1.0/_data.yaml:63]: @JM-GA Is this link useful? I see that a lot of product pages provide this link. I am guessing it's an API endpoint that can be used to retrieve GeoMAD data

Comment (JM-GA) [reply docs/data/old-version/dea-geometric-median-and-median-absolute-deviation-landsat-3.1.0/_data.yaml:63]: yep this is the OWS link that is required to add web services

**Comment by benji-glitsos-ga** [comment]
@JM-GA I've made just a few updates to this branch.

* Updated the old version page (3.1.0) with the latest changes from the main branch of the repo - https://pr-149-preview.khpreview.dea.ga.gov.au/data/old-version/dea-geometric-median-and-median-absolute-deviation-landsat-3.1.0/
* Wrote a Tech Alert entry - https://pr-149-preview.khpreview.dea.ga.gov.au/tech-alerts-changelog/#version-4-0-0-of-dea-geometric-median-and-median-absolute-deviation-released

**Review by benji-glitsos-ga — APPROVED** [review]

**Pull request merged**